app/controllers/api/v1/reviews_controller.rb

```diff
@@ -9,6 +9,10 @@ def create
     render json: new_review, adapter: :json
   end
 
+  def index
+    render json: Game.find(params[:game_id]).reviews.order("created_at DESC"), adapter: :json
+  end
+
```

Comment:
adapter: :json Should be specified in your Rails config, as per the serializer article from week 6. Please remove all mention from controllers and add there!